packages/legacy/src/Mappings/Mappings.tsx

```diff
@@ -50,7 +50,7 @@ export const Mappings: React.FunctionComponent<IMappingsProps> = ({
           <EmptyStateBody>
             {mappingType === MappingType.Network
               ? 'Map source provider networks to target provider networks.'
-              : 'Map source provider datastores or storage domains or volume types to target provider storage classes.'}
+              : 'Map source provider datastores or storage domains or volume types or storage classes to target provider storage classes.'}
```

Comment:
```suggestion
              : 'Map storage domains, datastores, volume types, or storage classes from the source provider to the storage classes of the target provider.'}
```

can we do something with less `or` s ?